docs/api/en/objects/Mesh.html

```diff
@@ -74,8 +74,15 @@ <h3>[property:Object morphTargetDictionary]</h3>
 		<h2>Methods</h2>
 		<p>See the base [page:Object3D] class for common methods.</p>
 
-		<h3>[method:Mesh clone]()</h3>
-		<p>Returns a clone of this [name] object and its descendants.</p>
+		<h3>
```

Comment:
I think it's better to completely remove this method since `clone()` is already documented in `Obejct3D`.